marbot.yml

```diff
@@ -1407,6 +1413,42 @@ Resources:
       Targets:
       - Arn: !Ref Topic
         Id: marbot
+  ECSSpotFailureEvent:
+    Condition: ECSSpotNotificationsEnabled
+    DependsOn: TopicEndpointSubscription
+    Type: 'AWS::Events::Rule'
+    Properties:
+      Description: 'ECS Spot failure. (created by marbot)'
+      EventPattern:
+        source:
+        - 'aws.ecs'
+        'detail-type':
+        - 'ECS Service Action'
+        detail:
+          eventName:
+          - SERVICE_TASK_PLACEMENT_FAILURE
```

Comment:
The docs mention another attirubte as well?
```
"reason": [
      "RESOURCE:FARGATE"
    ]
```

Reply:
✅